src/collective/sortedlisting/browser/results.pt

```diff
@@ -4,7 +4,7 @@
     there are no html and body tags
 </tal:comment>
 <div tal:define="results context;
-                 limited_results python:context[:25];
+                 limited_results python:context[:50];
```

Comment:
Can you use the same limit as in

https://github.com/collective/collective.sortedlisting/blob/master/src/collective/sortedlisting/browser/querybuilder.py#L18

This would result in subclassing `plone.app.querystring.querybuilder.ContentListingView` to set the limit as class-attribute there but I think it is worth the effort.